docs/user/resources/04-70-10-aws-vpc-peering.md

```diff
@@ -0,0 +1,65 @@
+## AWS Vpc Peering
+
+
+The `awsvpcpeering.cloud-resources.kyma-project.io` custom resource (CR) specifies the virtual network peering between
+Kyma and the remote AWS Virtual Private Cloud (VPC) network. Virtual network peering is only possible within the networks
+of the same cloud provider.
+
+Once an AwsVpcPeering CR is created and reconciled, the Cloud Manager controller first creates a virtual network peering
+connection in the Virtual Private Cloud (VPC) network of the Kyma cluster in the underlying cloud provider and accepts
+VPC peering connection in the remote cloud provider account.
+
+You must create CloudManagerPeeringRole and authorize Cloud Manager principal to assume that role in the remote cloud provider 
+account to accept VPC peering connection. Assign the following permissions to CloudManagerPeeringRole in the 
+remote account:
+* ec2:AcceptVpcPeeringConnection
+* ec2:DescribeVpcs
+* ec2:DescribeVpcPeeringConnections
+* ec2:DescribeRouteTables
+* ec2:CreateRoute
+* ec2:CreateTags
+
+AwsVpcPeering can be deleted at any time but the VPC peering connection in the remote account must be deleted
+manually.
```

Comment:
In my opinion, this should be written in a way to reflect that the kyma-side cloud provider peering is deleted as part of AwsVpcPeering deletion, but remote one is left hanging, and must be deleted manually.